discovery-infra/update_assisted_service_cm.py

```diff
@@ -56,11 +75,31 @@ def _get_relevant_envs():
     return data
 
 
+def update_requirements(requirements_json):
+    if requirements_json == "" or requirements_json == "REPLACE_HW_VALIDATOR_REQUIREMENTS":
+        return json.dumps(REQUIREMENTS)
+    try:
+        requirements = json.loads(requirements_json)
+        for version_requirements in requirements:
+            if version_requirements["version"] == "default":
+                version_requirements["master"] = MASTER_REQUIREMENTS
+                version_requirements["worker"] = WORKER_REQUIREMENTS
+    except json.JSONDecodeError:
+        raise
```

Comment:
It's a no-op
Meaning you can remove the complete wrapping of try-except and it will do the same